hhs_hosp/delphi_hhs/run.py

```diff
@@ -98,7 +98,7 @@ def run_module(params):
     all_columns = pd.concat(dfs)
     geo_mapper = GeoMapper()
     stats = []
-
+    all_columns=geo_mapper.add_population_column(all_columns, "state_code",geocode_col="state")
```

Comment:
step through this in a debugger -- before this line, all_columns has a pile of data in it; after this line, it is empty